modules/social_features/social_branding/social_branding.api.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/**
+ * @file
+ * Hooks provided by the Social Branding module.
+ */
+
+/**
+ * @addtogroup hooks
+ * @{
+ */
+
+/**
+ * Alter the list of preferred features.
+ *
+ * @param array $preferred_features
+ *   An array with the following key/value pairs:
+ *   - weight: The weight of the preferred feature.
+ *   - name: The machine name of the preferred feature.
+ *
+ * @ingroup social_branding_api
+ */
+function hook_social_branding_preferred_features_alter(array &$preferred_features) {
```

Comment:
Looks fine? I tried to look to use a base hook with weight and feature_name and an alter hook on top of that as you commented but I didn't find a good example.

Reply:
A good example can be seen in `hook_language_types_info` and `hook_language_types_info_alter`. I think we can nearly copy paste that (while adjusting the wording to our use-case of course) ;-) 

We should also strive to make the .api.php file something that can be copy pasted, so we need an illustrative function body here :) 

Let me know if you need help.